scss/main.scss

```diff
@@ -103,58 +109,29 @@ body {
     bottom: $home-button-position;
     right: $home-button-position;
     border-radius: 50%;
-    text-align: center;
-
-    opacity: 0;
-
 
+    .link {
```

Comment:
i nagle BEM się urywa(`aside__link`?). Jakbym wyniosł ten selektor poza klase. Wiem, że to w sumie to nie tyczy tego miejsca, ale ogólnie zbyt usilne zagnieżdżanie w scss prowadzi do nieczytelności i do niereużywalności.